### PR TITLE
Adding contribution guidelines

### PR DESCRIPTION
**Pull request by flamecoals**

### CODE REVIEW

File: web/docs/Contributing.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+There is a `GameStatus.IN_DEVELOPMENT` flag on the game definition that will hide the game from normal users, but will still show it for developers and allow you to use direct links for playtesters.
+If you are in the flow and does not want to stop your work for creating small PRs, you can use [git rebasing](https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits) and multiple branches to break it down in smaller chunks later.
+
+# Be careful with intellectual property.
+
+Even though we are not for profit, and our intentions are good - popularize board games and give more play opportunities beyond the physical experience -, we need to be careful in not infringing the intelectual property of others.
+In no hypothesis we can use games art, name, or rules copy without written consent from the copyright owners. Also, we cannot develop games if they have a patent on their rules (rare, but possible).
```
Comment (jasonharrison) [inline web/docs/Contributing.stories.mdx:19]: "In no hypothesis" sounds weird

File: web/docs/NewGame.stories.mdx [hunk 2]
```diff
@@ -163,4 +164,4 @@ Done! You can run now `yarn run dev` and you should be able to see your new game
 
 There are great boardgame.io [tutorials and documentation](https://boardgame.io/#/tutorial); you can follow any of them to get started on how to create your first game. Also, feel free to look into other FBG games and see how they are implemented. Finally, if you get stuck with some issue, send a message to our comunity! We will do our best to help you out :).
 
-When you have your game somewhat working, feel free to open a PR to our GitHub repo. If it is still a work-in-progress, just add "WIP" to the title. We can help you out and review your implementation when you have it working.
+Now, it is time to send us your game for review ! Please send PRs even if the game is still in development, we prefer small PRs over large ones. For more information, check the "Contributing" documentation.
```
Comment (jasonharrison) [inline web/docs/NewGame.stories.mdx:167]: Space after review and !

File: web/docs/Contributing.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+3. Implement the rules until all unit tests passes.
+4. Implement the User Interface / React components, and some unit tests for them.
+
+You can do all of these 4 steps for the whole game or only for subparts until complete.
+
+# Make sure your game works well on Mobile devices
+
+Nowadays, most people use their mobile phones more than their laptops or desktops, specially for leisure.
+We can expect this trend to continue over time, as young generations are growing up in this mobile-first world.
+Most other boardgame projects and websites out there do not work well on mobile, and since we started FBG we wanted this to be a differential.
```
Comment (jasonharrison) [inline web/docs/Contributing.stories.mdx:49]: "A differential" sounds awkward to me

**Review by jasonharrison — APPROVED** [review]

File: web/docs/Contributing.stories.mdx (hunk 1 above)
Comment (flamecoals) [reply web/docs/Contributing.stories.mdx:19]: Done.

File: web/docs/Contributing.stories.mdx (hunk 3 above)
Comment (flamecoals) [reply web/docs/Contributing.stories.mdx:49]: Done.

**Pull request merged**